debezium-core/src/main/java/io/debezium/jdbc/JdbcValueConverters.java

```diff
@@ -674,6 +680,78 @@ protected Object convertDateToEpochDays(Column column, Field fieldDefn, Object d
         });
     }
 
+    /**
+     * Converts a value object of various date-related types to its representation as a UTC ISO 8601 string.
+     * <p>
+     * This method attempts to convert the provided data object, which can be of types like
+     * {@link java.sql.Date}, {@link java.util.Date}, {@link java.time.LocalDate}, or
+     * {@link java.time.LocalDateTime}, to a String representing the date in UTC ISO 8601 format (e.g., "2023-11-21Z").
+     *
+     * @param column the column definition describing the {@code data} value; never null
+     * @param fieldDefn the field definition; never null
+     * @param data the data object to be converted into a UTC ISO 8601 String; can be null
+     * @return the converted UTC ISO 8601 String representation of the date,
+     * or "1970-01-01Z", the fallback value, if the column is non-null and no default value is provided
+     * @throws IllegalArgumentException if the conversion fails and the column does not allow nulls
+     */
+    protected Object convertDateToUtcIsoString(Column column, Field fieldDefn, Object data) {
+        return convertValue(column, fieldDefn, data, "1970-01-01Z", (r) -> {
+            try {
+                r.deliver(IsoDate.toIsoString(data, adjuster));
+            }
+            catch (IllegalArgumentException e) {
+                logger.warn("Unexpected JDBC DATE value for field {} with schema {}: class={}, value={}", fieldDefn.name(),
+                        fieldDefn.schema(), data.getClass(), data);
+            }
+        });
+    }
+
+    /**
+     * Converts a value object of various time-related types to its representation as a UTC ISO 8601 string.
+     * <p>
+     * This method handles conversion of time-related values such as {@link java.sql.Time},
+     * {@link java.util.Date}, {@link java.time.LocalTime}, and {@link java.time.LocalDateTime} to a
+     * UTC ISO 8601 string format (e.g., "00:00:00Z").
+
+     * @param column the column definition describing the data value; never null
+     * @param fieldDefn the field definition; never null
+     * @param data the data object to be converted; can be null
+     * @return the converted UTC ISO 8601 string representation of the time, or "00:00:00Z" if necessary
+     */
+    protected Object convertTimeToUtcIsoString(Column column, Field fieldDefn, Object data) {
+        // epoch is the fallback value
+        return convertValue(column, fieldDefn, data, "00:00:00Z", (r) -> {
+            try {
+                r.deliver(IsoTime.toIsoString(data, supportsLargeTimeValues()));
+            }
+            catch (IllegalArgumentException e) {
+                e.printStackTrace();
```

Comment:
Either use `WAR` log or ignore the message.

Reply:
changed to `logger.warn`